packages/blade/src/components/SideNav/_decisions/decisions.md

```diff
@@ -0,0 +1,418 @@
+| rel         | anchor tag rel attribute [rel - MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#rel)          | AnchorRelType                                                       | target === ' \_blank ' ? ' noreferrer noopener ' : undefined |
+| onClick     | Click handler on item                                                                                                        | (e: React.MouseEvent) => void                                       |                                                              |
+| icon        | Blade's Icon Component                                                                                                       | IconComponent                                                       |                                                              |
+| trailing    | Trailing Slot of Item. Can be used for adding Quick Shortcut Button, Trailing Text                                           | JSX                                                                 |                                                              |
+| titleSuffix | Slot after the title to add Badge, Counter                                                                                   | JSX                                                                 |                                                              |
+| ref         | Forwards the given ref. This also allows it to be a trigger for Tooltip                                                      | idk man ref types are hard.<br/> I'll figure out while implementing |                                                              |
+| children    | SideNavLink children slot. Items inside children turn into next level item with parent as a trigger                          | JSX                                                                 | undefined                                                    |
+
+#### Example Nav Links
+
+<table>
+
+<tr><td>Code</td><td>Preview</td></tr>
+
+<tr>
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<SideNavLink
+  icon={BillIcon}
+  title="Vendor Payments"
+  href="/vendor-payments"
+/>
+```
+
+</td>
+<td><img src="navlink-eg-1.png" width="200px" /></td>
+</tr>
+
+<tr>
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<SideNavLink
+  icon={PayoutIcon}
+  title="Create Payouts"
+  href="/payouts"
+  trailing={
+    <Button
+      href="/payouts/create"
+      icon={PlusIcon}
+      size="xsmall"
+      variant="tertiary"
+    />
+  }
+/>
+```
+
+</td>
+<td><img src="navlink-eg-2.png" width="200px" /></td>
+</tr>
+
+<tr>
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<SideNavLink
+  icon={LayoutIcon}
+  title="L1 Item Name"
+  href="/new-item-link"
+  titleSuffix={
+    <Badge color="positive">New</Badge>
+  }
+/>
+```
+
+</td>
+<td><img src="navlink-eg-3.png" width="200px" /></td>
+</tr>
+
+<tr>
+
+<td>
+
+<!-- prettier-ignore -->
+```jsx
+<Tooltip content="Action Name (Cmd + P)">
```

Comment:
is it also possible to have an info icon and show the tooltip on that? can we add an example of that?

Reply:
also shall we not have tooltip on the entire nav link and have on info icon? because a11y?

Reply:
I think it will be there on overall item (similar to our Tooltip with Button example). Because tooltip represents what happens on click of item.

Anurag suggested having tooltip inside SideNavLink component as first-class feature so will explore that